### PR TITLE
update notification queue message model

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1096

### Change description ###
Update Notification Message to add `container` and remove `jurisdiction` and `po_box` as Blob router doesn't have these values.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java [hunk 1]
```diff
@@ -7,11 +7,8 @@
     @JsonProperty("zip_file_name")
     public final String zipFileName;
 
-    @JsonProperty("jurisdiction")
-    public final String jurisdiction;
-
-    @JsonProperty("po_box")
-    public final String poBox;
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java]: are those 2 not required by the notification microservice?

File: src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java [hunk 2]
```diff
@@ -7,11 +7,8 @@
...
+    @JsonProperty("container")
+    public final String container;
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java]: will notification microservice use this field?

File: src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java (hunk 1 above)
Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java]: no, they are optional

File: src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java (hunk 2 above)
Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/model/NotificationMsg.java]: yes, container is added in https://github.com/hmcts/reform-scan-notification-service/pull/62

File: src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/NotificationsPublisher.java [hunk 3]
```diff
@@ -42,20 +42,18 @@ public void publish(NotificationMsg notificationMsg) {
...
-                notificationMsg.poBox,
+                notificationMsg.container,
                 notificationMsg.errorCode
             );
         } catch (Exception ex) {
             throw new NotificationsPublishingException(
                 String.format(
                     "An error occurred when trying to publish notification for "
-                        + "Blob name: %s, Jurisdiction: %s, PO Box: %s",
+                        + "File name: %s, Container: %s",
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/NotificationsPublisher.java]: perhaps fits in one line now?

**Pull request merged**